neqo-transport/src/fc.rs

```diff
@@ -213,6 +240,10 @@ where
         }
     }
 
+    pub fn add_retired(&mut self, retired: u64) {
```

Comment:
As I've mentioned, you might be able to replace this with an accessor for self.retired instead (though if you move the code as I suggest, you can just access the variable directly).